src/main/resources/digital/slovensko/autogram/ui/gui/idsk.css

```diff
@@ -564,6 +576,31 @@ TextFlow.autogram-body-s {
   -fx-background-color: -autogram-border-colour;
 }
 
+
+.autogram-tag {
+  -fx-border-width: 2px;
+  -fx-padding: 0.1875em 0.25em;
+}
+
+.autogram-tag-valid {
+  -fx-background-color: #cce2d8;
+  -fx-border-color: #cce2d8;
+}
+
+.autogram-tag-invalid {
+  -fx-background-color: #f8d7da;
+  -fx-border-color: #f8d7da;
+}
+
+.autogram-tag-valid--text {
+  -fx-text-fill: #005a30;
+  -fx-fill: #005a30;
+}
+
+.autogram-tag-invalid--text {
+  -fx-text-fill: #721c24;
+  -fx-fill: #721c24;
```

Comment:
Farby potom vytiahnime do konstant

Reply:
Ďalej pôjdem robiť GUI, tak pri tej príležitosti to povyťahujem.